lib/hanami/logger.rb

```diff
@@ -145,7 +145,7 @@ def call(severity, time, _progname, msg)
         _format({
           app:      @application_name,
           severity: severity,
-          time:     time.utc
+          time:     time.utc.to_datetime.rfc3339
```

Comment:
The problem here is the creation of intermediate objects:

```ruby
irb(main):003:0> now = Time.now
=> 2017-01-05 18:05:09 +0100
irb(main):004:0> now.object_id
=> 70259928409580
irb(main):005:0> now.utc.object_id
=> 70259928409580
irb(main):006:0> now.utc.object_id
=> 70259928409580
irb(main):007:0> now.utc.to_datetime.object_id
=> 70259928246620
irb(main):008:0> now.utc.to_datetime.object_id
=> 70259928186200
irb(main):009:0> now.utc.to_datetime.rfc3339.object_id
=> 70259928152180
irb(main):010:0> now.utc.to_datetime.rfc3339.object_id
=> 70259928064880
```

As you can see, when `time` is converted in UTC, it's still keeping the same Object ID, which is good. But for `to_datetime` and `rfc3339`, it creates two new objects.

If we want to keep RFC 3339, we should avoid to create the intermediary `DateTime` object.